solidity/ecdsa/deploy/00_resolve_random_beacon.ts

```diff
@@ -10,12 +10,19 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
 
   const RandomBeacon = await deployments.getOrNull("RandomBeacon")
 
+  const isRandomBeaconNeeded = function () {
+    if (hre.network.tags.chaosnet) {
+      return false
+    }
+    if (!hre.network.tags.allowStubs) {
+      return true
+    }
+    return (hre.network.config as HardhatNetworkConfig)?.forking?.enabled
+  }
+
   if (RandomBeacon && helpers.address.isValid(RandomBeacon.address)) {
     log(`using existing RandomBeacon at ${RandomBeacon.address}`)
-  } else if (
-    !hre.network.tags.allowStubs ||
-    (hre.network.config as HardhatNetworkConfig)?.forking?.enabled
-  ) {
+  } else if (isRandomBeaconNeeded()) {
     throw new Error("deployed RandomBeacon contract not found")
```

Comment:
This file has `func.tags = ["RandomBeacon"]` and no one uses this tag so we could actually... delete it?

Reply:
After recent changes the tag is used again [here](https://github.com/keep-network/keep-core/blob/c58aa96ec1da823d5345e588fa7f6202e6c24ad6/solidity/ecdsa/deploy/03_deploy_wallet_registry.ts#L77).